### PR TITLE
Jeff/higher lower

### PR DESCRIPTION
**Pull request by jeffberdin**
I push my recent commit.

### CODE REVIEW

File: higher_lower_project/main.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    clear()
+    print(logo)
+    if is_correct:
+      score += 1
+      print(f"You're right! Current score: {score}.")
+    else:
+      game_should_continue = False
+      print(f"Sorry, that's wrong. Final score: {score}")
+
+game()
```
Comment (grantelledge) [inline higher_lower_project/main.py:59]: I'll just ask a reflective question here at the end, no need to respond -- do you feel like you understand how the dictionary was used here? Comfortable enough to replicate the concept?

Comment (jeffberdin) [reply higher_lower_project/main.py:59]: dictionary in Python is a collection of data which is unordered and mutable. It is used to store unrelated data types but data that is related as a real-world entity.

Comment (grantelledge) [reply higher_lower_project/main.py:59]: Mind hitting a few of those bigger ideas in your own words (unordered, mutable, unrelated types, related entities)?

**Comment by jeffberdin** [comment]
data = [
{
'name': 'Cristiano Ronaldo',
'follower_count': 215,
'description': 'Footballer',
'country': 'Portugal'
},
{
'name': 'Ariana Grande',
'follower_count': 183,
'description': 'Musician and actress',
'country': 'United States'
},
{
'name': 'Dwayne Johnson',
'follower_count': 181,
'description': 'Actor and professional wrestler',
'country': 'United States'
},
]
This is the example of 'dictionary'.
the data of social media influencers stored in the variable 'data'.

**Pull request merged**